app/services/editions/db/IssueQueries.scala

```diff
@@ -13,7 +13,8 @@ trait IssueQueries {
   def insertIssue(
                    name: String,
                    skeleton: EditionsIssueSkeleton,
-                   user: User
+                   user: User,
+                   now: OffsetDateTime
```

Comment:
It feels a little odd to pass `now` as a param. Would renaming it to `date` or defaulting it to `DateTime.now` be better?

Reply:
Update: From reading the tests, this makes a lot of sense.